bitbots_vision/bitbots_vision/vision_modules/ros_utils.py

```diff
@@ -112,8 +113,7 @@ def build_goal_post_array_msg(header, goal_post_msgs):
     # Create goalposts msg
     goal_posts_msg = GoalpostArray()
     # Add header
-    goal_posts_msg.header.frame_id = header.frame_id
-    goal_posts_msg.header.stamp = header.stamp
+    goal_posts_msg.header = header
```

Comment:
The header contains more than the two things. It should be fine, but it is not equivalent.

Reply:
I thought so too, but I checked this yesterday:
```
$ ros2 interface show std_msgs/msg/Header   
# Standard metadata for higher-level stamped data types.
# This is generally used to communicate timestamped data
# in a particular coordinate frame.

# Two-integer timestamp that is expressed as seconds and nanoseconds.
builtin_interfaces/Time stamp
	int32 sec
	uint32 nanosec

# Transform frame with which this data is associated.
string frame_id
```